code/modules/cargo/packs/livestock.dm

```diff
@@ -52,7 +52,7 @@
 	desc = "The chicken goes bwaak!"
 	cost = CARGO_CRATE_VALUE * 4
 	access_view = ACCESS_KITCHEN
-	contains = list( /mob/living/simple_animal/chick)
+	contains = list( /mob/living/basic/chick)
```

Comment:
```suggestion
	contains = list(/mob/living/basic/chick)
```

Reply:
yeah this was done on an unpushed commit sorry